### PR TITLE
Add communication log for RDMA(verbs)

### PR DESCRIPTION
**Pull request by sj6077**
**Major changes:**
-  When tracing timeline, RDMA communication log is included after this PR.

**Minor changes to note:**
-  add PR template 

**Tests for the changes:**
- You can check 'RecvTensor' in RunMetata after tracing a timeline using grpc+verbs communication.

**Other comments:**

### CODE REVIEW

File: tensorflow/contrib/verbs/rdma_rendezvous_mgr.cc [hunk 1]
```diff
@@ -74,8 +77,33 @@ void RdmaRemoteRendezvous::RecvFromRemoteAsync(
...
+  DoneCallback wrapper_done;
+  const DoneCallback* cb_to_use;
+  if(!logging_active) { 
+    cb_to_use = &done; //No additional work to do, so just use done directly
+  } else {
+     int64 step_id = step_id_;
+     wrapper_done = [this, logger, step_id, parsed, done, start_usec] (const Status& s, const Args& send_args, const Args& recv_args,
+  	                                                               const Tensor& recv_tensor, const bool is_dead) {
+      
+      if (logger->LoggingActive()) {
```
Comment (hjp615) [inline tensorflow/contrib/verbs/rdma_rendezvous_mgr.cc]: Is this line necessary? I think `else` in line 88 checks this condition.

File: tensorflow/contrib/verbs/rdma_rendezvous_mgr.cc [hunk 2]
```diff
@@ -74,8 +77,33 @@ void RdmaRemoteRendezvous::RecvFromRemoteAsync(
...
+  // Type-specialized logging for this method.
+  WorkerCacheLogger* logger = rdma_mgr_->GetLogger();
+
+  bool logging_active = logger->LoggingActive();
+  DoneCallback wrapper_done;
+  const DoneCallback* cb_to_use;
+  if(!logging_active) { 
+    cb_to_use = &done; //No additional work to do, so just use done directly
+  } else {
+     int64 step_id = step_id_;
```
Comment (hjp615) [inline tensorflow/contrib/verbs/rdma_rendezvous_mgr.cc]: Fix indentation.

**Review by hjp615 — CHANGES_REQUESTED** [review]
@sj6077 Thanks. I left some minor comments. Could you please check?

File: tensorflow/contrib/verbs/rdma_rendezvous_mgr.cc (hunk 1 above)
Comment (sj6077) [reply tensorflow/contrib/verbs/rdma_rendezvous_mgr.cc]: This is callback function for an asynchronous call, so I think it is for a recheck the logging activation.

**Comment by sj6077** [comment]
@hjp615 I addressed your comments. Can you check and merge it?

**Comment by hjp615** [comment]
Thanks! LGTM. I'll merge it.

**Review by hjp615 — APPROVED** [review]

**Pull request merged**